bundles/org.openhab.binding.miele/src/main/resources/OH-INF/thing/fridgefreezer.xml

```diff
@@ -19,10 +19,22 @@
 			<channel id="fridgestate" typeId="fridgestate"/>
 			<channel id="supercool" typeId="supercool"/>
 			<channel id="superfreeze" typeId="superfreeze"/>
-			<channel id="freezercurrent" typeId="freezercurrent"/>
-			<channel id="freezertarget" typeId="freezertarget"/>
-			<channel id="fridgecurrent" typeId="fridgecurrent"/>
-			<channel id="fridgetarget" typeId="fridgetarget"/>
+			<channel id="freezercurrent" typeId="temperature">
+				<label>Current Temperature</label>
+				<description>Current temperature in the freezer compartment</description>
+			</channel>
+			<channel id="freezertarget" typeId="temperature">
+				<label>Target Temperature</label>
+				<description>Target temperature to be reached by the freezer compartment</description>
+			</channel>
+			<channel id="fridgecurrent" typeId="temperature">
+				<label>Current Temperature</label>
+				<description>Current temperature in the fridge compartment</description>
+			</channel>
+			<channel id="fridgetarget" typeId="temperature">
+				<label>Target Temperature</label>
+				<description>Target temperature to be reached by the fridge compartment</description>
+			</channel>
```

Comment:
Same remark here.
But here, it makes sense to have only one common "targetTemperature channel type for 2 different channels."

Reply:
You convinced me to split into current and target for reuse of same label - fixed.